diff_drive_controller/src/diff_drive_controller.cpp

```diff
@@ -689,8 +689,9 @@ controller_interface::CallbackReturn DiffDriveController::configure_side(
 
 bool DiffDriveController::on_set_chained_mode(bool chained_mode)
 {
-  // Always accept switch to/from chained mode (without linting type-cast error)
-  return true || chained_mode;
+  // Fix to adhere to CppLint standards
```

Comment:
A motor reference isn't really adhering to cppkint standards ;)

Reply:
You can achieve the same by commenting out the name of the argument. It tricks the parser and also the compiler. We already use that approach in the codebase, just look for "/*" in ros2_control

Reply:
No, I don't think so. See https://github.com/ros-controls/ros2_controllers/issues/1491 and the linked cpplint issue.